src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentDictionary.cs

```diff
@@ -1786,14 +1780,14 @@ private void GrowTable(Tables tables)
 
                 // Add more locks
                 if (_growLockArray && tables._locks.Length < MaxLockNumber)
-                {
-                    newLocks = new object[tables._locks.Length * 2];
-                    Array.Copy(tables._locks, 0, newLocks, 0, tables._locks.Length);
-                    for (int i = tables._locks.Length; i < newLocks.Length; i++)
                     {
-                        newLocks[i] = new object();
+                        newLocks = new object[tables._locks.Length * 2];
+                        Array.Copy(tables._locks, 0, newLocks, 0, tables._locks.Length);
+                        for (int i = tables._locks.Length; i < newLocks.Length; i++)
+                        {
+                            newLocks[i] = new object();
+                        }
```

Comment:
Why did this get indented?

Reply:
My bad. Mixing up changes. Will fix that. I'm sorry.